internal/services/zero_trust_access_service_token/resource_test.go

```diff
@@ -54,7 +54,7 @@ func TestAccCloudflareAccessServiceToken_Basic(t *testing.T) {
 					resource.TestCheckResourceAttr(name, consts.AccountIDSchemaKey, accountID),
 					resource.TestCheckResourceAttr(name, "name", resourceName+"-updated"),
 					resource.TestCheckResourceAttrSet(name, "client_id"),
-					resource.TestCheckResourceAttrSet(name, "client_secret"),
+					resource.TestCheckNoResourceAttr(name, "client_secret"),
```

Comment:
i think this is a regression; we need to have the client_secret in the state file which here looks to be failing.